charter.html

```diff
@@ -0,0 +1,330 @@
+      browser (or similar user agent) should be proposed in other standards
+      bodies (IETF, etc).
+    </p>
+    <h2 id="meetings">
+      Meetings
+    </h2>
+    <p>
+      Meetings will be held biweekly on {TODO: INSERT DATE}, lasting for one
+      hour. Meetings will be run by one of the Chairs and will request scribes
+      to take meeting notes, which will be posted in the
+      <a href="https://github.com/antifraudcg/meetings">Meetings</a> repository
+       after the meeting. Any action items from the meeting will also be posted
+       to the Community Group mailing list to bring them to the attention of
+       people who did not attend.
+    </p>
+    <p>
+      Infrequently, face-to-face meetings will be held for the Anti-Fraud
+      Community Group, allowing for more collaboration and iteration between
+      Community Group members.
+    </p>
+    <h2 id="deliverables">
+      Deliverables
+    </h2>
+    <h3 id="work-items">
+      Current Work Items
+    </h3>
+    <p>
+      There are currently no work items that have been adopted by the Community
+      Group. See <a href="#proposals">Proposals</a> for the process of adopting
+      a proposal into the Community Group.
+    </p>
+    <p>
+      Once a work item is ready for standardization and there is
+      <a href="#decision">consensus</a> that the work item is ready to
+      be migrated to a standards body, the Chairs will collaborate with the
+      Community Group to prepare a Community Group Report on the item, and with
+      the Editors of the item to forward it and the Report to the appropriate
+      standards body.
+    </p>
+    <h3 id="proposals">
+      Proposals
+    </h3>
+    <p>
+      Anyone can submit proposals to the proposals repository. If there is
+      substantial documentation/work, the proposal authors can request the
+      Chairs set-up a dedicated repository for a proposal.
+    </p>
+    <p>
+      A proposal can be withdrawn by its authors, or closed if there's
+      consensus that the proposal is out of scope or the number of authors has
+      dropped to zero. The Chairs will check-in with active proposals to ensure
+      the set of active authors is kept up-to-date.
+    </p>
+    <p>
+      Generally proposals should include an <i>Explainer</i> document describing
+      the changes to the web platform and impact of the proposal.
+    </p>
+    <p>
+      Work items are proposals that can lead to interoperable standards/APIs.
+      Once a proposal is in a state for the community group to work on, it
+      should be proposed for adoption as a work item. To be adopted as a work
+      item, a proposal should be sent out to the CG mailing list, and there
+      must be at least two supporters of the proposal. For work items involving
+      web-exposed APIs, at least one supporter must be a browser vendor (as an
+      indication of interest in implementation).
```

Comment:
As raised on the call, I don't think this explicit browser vendor requirement is necessary, given that there is a more general requirement in the next paragraph for sufficient likelihood of implementation.